### ISSUE TITLE
Add a Planner use example to GSG story

### ISSUE THREAD

**Issue by rkratky** [fabric8io/fabric8-online-docs#235]
Expand at least one GSG story with an example of Planner use. Link to reference info in the UG as needed.

Could look something like this:

1. Start by planning a project (?)
2. Create and assign WI(s)
3. Pick up WI(s) and start coding
4. (example of an urgent WI to curb resource use?)
5. Finish work, indicate as much in the Planner

**Comment by Preeticp** [comment]
I can help on this.

**Comment by rkratky** [comment]
@Preeticp Do you mean you want to take it, or just offering help?

**Comment by Preeticp** [comment]
I can take it up.

**Comment by Preeticp** [comment]
Do we have more information on point 4-(example of an urgent WI to curb resource use?)

**Comment by rkratky** [comment]
> Do we have more information on point 4-(example of an urgent WI to curb resource use?)

Not really, just an idea.

**Comment by Preeticp** [comment]
@rkratky unassigning myself from this and adding help wanted. I see quite a few things on my plate apart from the issues here such as the Planner UX work coming up.

**Comment by mishaone** [comment]
We could create a work item to track the optimizing resources part we already have, if we want to adapt the existing content in the GS doc. Is that what you had in mind, @rkratky. If so, I can probably get this done fairly quickly.

**Comment by rkratky** [comment]
@mishaone Good idea. The reason I included that suggestion was to make the Planner workflow a tiny bit more involved than just:

1. create a WI
2. do the task
3. close the WI

So, I'd like to interject some additional task that reacts to what is being done for the first task. The resource optimization idea is ideal.

**Comment by mishaone** [comment]
Cool. In that case, I can do it fairly easily. Assigning to myself.

**Comment by Preeticp** [comment]
@mishaone I can take this one up. Is it alright with you? Have you already begun work on this one?

**Comment by mishaone** [comment]
Not yet, @preeticp. Feel free to take it. :)

**Comment by Preeticp** [comment]
WIP now
